### PR TITLE
add deserialization cache tunable to host recommendations

### PR DESCRIPTION
**Pull request by sjenning**
@ncdc @smarterclayton @adellape

### CODE REVIEW

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -136,6 +136,19 @@ RAM.
 When planning an environment with multiple masters, a minimum of three etcd
 hosts as well as a load-balancer between the master hosts, is required.
 
+The Openshift master caches deserialized versions of resources fairly
```
Comment (ncdc) [inline install_config/install/prerequisites.adoc]: OpenShift

File: install_config/install/prerequisites.adoc [hunk 2]
```diff
@@ -136,6 +136,19 @@ RAM.
...
+aggressively to ease CPU load.  However, in smaller clusters of less than 1000
+pods, this cache can waste a lot of memory for negligible CPU load reduction.
```
Comment (ncdc) [inline install_config/install/prerequisites.adoc]: It's not just pods, is it? What about all the other resources (DeploymentConfigs, Builds, etc)?

File: install_config/install/prerequisites.adoc [hunk 3]
```diff
@@ -136,6 +136,19 @@ RAM.
...
+The default cache size is 50000 entries which, depending on the size of your
```
Comment (ncdc) [inline install_config/install/prerequisites.adoc]: comma after entries

File: install_config/install/prerequisites.adoc (hunk 2 above)
Comment (sjenning) [inline install_config/install/prerequisites.adoc]: Yes, but I'm not sure how to say that concisely.  "Pods" was the measurement of cluster size used in the previous paragraph so I just used that.

**Comment by ncdc** [comment]
ping @adellape

**Comment by ahardin-rh** [comment]
@sjenning Thanks! Can you please give this a squash?

**Comment by sjenning** [comment]
@ahardin-rh rebased on latest master and squashed

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Installing -> Prerequisites]
|Added that the deserialization cache size can be reduced using a setting in *_master-config.yaml_*.
%

**Pull request merged**